npc/re/merchants/biosphere.txt

```diff
@@ -0,0 +1,252 @@
+				mes "It embodies the ecosystem of the grassland area. Unlike other habitats, you might think that it is a temperate environment, but living things grow better in such an environment.";
+				close2;
+				navigateto("ba_in01",325,56);
+				end;
+			case 4:
+				mes "[Admin beta]";
+				mes "Sample Environment - <FONT COLOR = 660099><B>Death</B></FONT>";
+				mes "This is a place that embodies the ecosystem of the area where the aura of death is present. There are many monsters who have gained more power at the border of death and life, so be careful.";
+				close2;
+				navigateto("ba_in01",339,56);
+				end;
+			case 5:
+				mes "[Admin beta]";
+				mes "Adventurous guests show great interest in powerful equipment.";
+				next;
+				mes "[Admin beta]";
+				mes "So, I've found a way to transform runes and equipment obtained from artificial habitats into powerful equipment that can't be found anywhere, so if you're interested, please visit it.";
+				close2;
+				navigateto("ba_in01",362,48);
+				end;
+			}
+		}
+		end;
+	case 2:
+		mes "[Admin beta]";
+		mes "The Large Bath Meditatio is the largest recreational facility in the Varmundt Mansion. The owner used to end the day here by relieving old fatigue.";
+		next;
+		mes "[Admin beta]";
+		mes "Varmundt showed great interest in the various effects of water and herbs, and gradually expanded the bath facilities in the mansion, and Meditatio has the current bath facilities.";
+		next;
+		mes "[Admin beta]";
+		mes "Varmundt provided bathing facilities to the guests staying at the mansion. The guests admired the fairly large bathing facilities and cherished them without hesitation.";
+		next;
+		mes "[Admin beta]";
+		mes "Meditatio has baths with unique nicknames. Among the guests, the place called the Fountain of Wisdom seems to be particularly popular.";
+		next;
+		select( "That's a nice place" );
+		mes "[Admin beta]";
+		mes "But... It looks like there's been a bit of a problem lately. Oh, that's right. It would be nice for an adventurer who is an expert in this field to visit.";
+		next;
+		mes "[Admin beta]";
+		mes "Go to the <NAVI>[gentleman]<INFO>ba_in01,384,38,0,101,0</INFO></NAVI> who manages it, and you will get directions.";
+		close;
+	}
+	end;
+
+OnTouch:
+	npctalk "Guest, do you need a guide inside the mansion?", "", bc_self;
+	end;
+}
+
+
+// Exchange npcs
+ba_in01,358,50,8	script	Omega#ms01	EP17_2_OMEGA_CLEANER,{
+	mes "[Omega]";
+	mes "Vrr-R-R-R-R-R-R-R-R";
+	next;
+	mes "[" + strcharinfo(0) + "]";
+	mes "It seems to convert the pieces of runes obtained inside the artificial habitat into complete runes.";";
+	close2;
+	callshop( "barter_biosphere_1" );
+	end;
+
+OnWalk1:
+	npctalk "Vrr-R-R-R-R-R-R";
+	unitstopwalk getnpcid(0), USW_FORCE_STOP;
+	unitwalk getnpcid(0),365,50, strnpcinfo(0) + "::OnWalk2";
+	end;
+OnWalk2:
+	unitstopwalk getnpcid(0), USW_FORCE_STOP;
+	unitwalk getnpcid(0),358,50, strnpcinfo(0) + "::OnWalk1";
+	end;
+OnInit:
+	npcspeed 500;	// todo check
```

Comment:
Im not sure but from my aegis i found this
```    NpcMovePoint "오메가#ms01" 900 365 50
    NpcMovePoint "오메가#ms01" 900 358 50```
365/358 and 50 is the location of movement
i guess 900 is the npcspeed, it seems to be correct.